remove_not_unique.py

```diff
@@ -0,0 +1,5 @@
+data = [1,2,1,2,1,2,3]
+
+for val in set(data):
+    if data.count(val) == 1:
+        data.remove(val)
```

Comment:
as a result there should be a function

Reply:
I have changed everything, as you suggested in the prev. task. 
Please, review changes